### ISSUE TITLE
Chrome App Migration?

### ISSUE THREAD

**Issue by bgiesing** [flackr/circ#414]
In case you haven't seen yet @flackr, Google is discontinuing Chrome apps on every platform except Chrome OS. Any plans for CIRC then? Google says devs should convert it to either Progressive Web Apps or native packaging with Electron/NW.js.

They mention this also:

> Q: My app uses the chrome sockets API to do networking. Can this be done with the open web?
> 
> You might be able to do what you need with WebSockets. However, to use this, you will likely need to change the remote end of your connection. **If that isn’t possible (e.g. you’re connecting over an established protocol like IRC or SSH), you’ll need some sort of native component.**

https://developers.chrome.com/apps/migration

**Comment by atomic1fire** [comment]
NW.js would probably be more likely, since they announced they have support for chrome apis

http://nwjs.io/blog/chrome-apps-support/

**Comment by caub** [comment]
seeing the code, and how it uses globals, written in coffesript, uses jquery, I think it wouldn't be bad to rewrite it

**Comment by Deide** [comment]
I've forked and rewritten it with ES6/babel. No tests though, and a bug or two, but it's operational.
However, I believe a new CIRC is being worked on anyway.

**Comment by flackr** [comment]
Yes, it's coming along slowly but if you want to see it there's a new version in development on the "server" branch. It currently requires running a nodejs server but I'll be adding support for running as a chrome packaged app so that it continues to work as a standalone application on chromebooks.
